openlibrary/templates/widget.html

```diff
@@ -26,24 +26,24 @@ <h2 class="author">
       $if book_status == "open" :
         $ public_url = canonical_url("/books/" + item["availability"]["openlibrary_edition"] + "/x/borrow")
         <div class="cta open" type="submit">
-          <a data-ol-link-track="Embed|Read"  target="_blank" href="$public_url" title="Read &quot;$book_title&quot;">Read</a>
+          <a data-ol-link-track="Embed|Read"  target="_blank" href="$public_url" title="$_('Read \"%(title)s\"', title=book_title)">$_('Read')</a>
 	</div>
       $elif book_status == "borrow_available" :
         $ borrow_url = canonical_url("/borrow/ia/" + item["availability"]["identifier"])
         <div class="cta available" type="submit">
-          <a data-ol-link-track="Embed|Borrow" target="_blank" href="$borrow_url" title="Borrow &quot;$book_title&quot;">Borrow</a>
+          <a data-ol-link-track="Embed|Borrow" target="_blank" href="$borrow_url" title="$_('Borrow \"%(title)s\"', title=book_title)">$_('Borrow')</a>
```

Comment:
Note to self test this; do the `"` need to be twice escaped? Or should we switch to using `'` in the html here?

Reply:
Tested; work as-is